cmake/test.cmake

```diff
@@ -43,6 +43,7 @@ endfunction()
 #                        that mayaUsd_add_test manages (and will append) are:
 #                            PATH
 #                            PYTHONPATH
+#                            PXR_USD_WINDOWS_DLL_PATH
```

Comment:
In PR #2867 a client added this env var to the .mod files. I've updated our testing environment to match.